### PR TITLE
Fix typo in Lesson 1 Notes.

### PR DESCRIPTION
**Pull request by nickivance**
Hi,
Sounds like "test set" was meant to be "training set", based on what I've read in the [Lesson 1 notebook jargon section] (https://github.com/fastai/fastbook/blob/master/01_intro.ipynb) and heard in the video.
Thanks!

### CODE REVIEW

**Comment by jph00** [comment]
Nope test set is what we meant!
